src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_acfr.yaml

```diff
@@ -37,8 +37,8 @@ interactions:
         {},\r\n    \"redisConfiguration\": {\r\n      \"maxclients\": \"1000\",\r\n
         \     \"maxmemory-reserved\": \"50\",\r\n      \"maxfragmentationmemory-reserved\":
         \"50\",\r\n      \"maxmemory-delta\": \"50\"\r\n    },\r\n    \"accessKeys\":
-        {\r\n      \"primaryKey\": \"xY3tVWeu5LlRK6sZrn7d02gX6kq9ZQhIRAzCaM0nhrI=\",\r\n
-        \     \"secondaryKey\": \"0hY15HHIXMKHUozFgVwnVTl8KPXELzE6zAzCaKQaLg8=\"\r\n
+        {\r\n      \"primaryKey\": \"abc\",\r\n
+        \     \"secondaryKey\": \"abc\"\r\n
```

Comment:
If YAML is recorded again, Credential Scan will fail again.

Reply:
Better way is to use replacer to replace credentials. Just removing credential to quick fix CI failure.

Reply:
It was previously manually replaced. That's why we hit this pitfall again: https://github.com/Azure/azure-cli/pull/19967#discussion_r776609439.